hack/boilerplate/boilerplate.py

```diff
@@ -133,6 +133,20 @@ def file_extension(filename):
                 "vendor", "test/e2e/generated/bindata.go", "hack/boilerplate/test",
                 "pkg/generated/bindata.go"]
 
+# list all the dirs that contain generated files
+generated_dirs = ['pkg/client/listers', 'pkg/client/clientset_generated', 'pkg/client/informers', 'k8s.io/sample-controller/pkg/client/listers',
```

Comment:
I am also against having another directory list. Note that we will also have this "DO NOT EDIT" line in generated files. Can't we just use that to detect generated files?

Reply:
yeah, it's a nice idea, but the comment in different files is also different, like [pkg/client/listers/admissionregistration/internalversion/expansion_generated.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/listers/admissionregistration/internalversion/expansion_generated.go)  and  [zz_generated.deepcopy.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/abac/zz_generated.deepcopy.go)

Reply:
How about make the comments all the same? then may be we can identify them easily

Reply:
With an updated gengo it will be uniform.

Reply:
https://github.com/kubernetes/gengo/pull/94

Reply:
OK, some files generated from script in hack maybe also need fixed.